indico/modules/events/surveys/templates/management/survey_submission.html

```diff
@@ -21,13 +21,26 @@
 {% endblock %}
 
 {% block content %}
-    <div class="i-box-content">
-        <dl class="i-data-list">
-            {% for answer in submission.answers -%}
-                <dt><strong>{{ answer.question.title }}</strong></dt>
-                <dd>{{ _render_answer(answer) }}</dd>
-            {%- endfor %}
-        </dl>
+    <div id="survey-submission" class="i-box-content">
+        {% for section in survey.sections if section.children -%}
```

Comment:
Hm, that should work - one extra query to get the sections but no looped queries...